packages/driver/src/multi-domain/commands.ts

```diff
@@ -14,7 +15,11 @@ export const handleCommands = (Cypress: Cypress.Cypress, cy: $Cy, specBridgeComm
     const id = command.get('id')
     const name = command.get('name')
 
-    specBridgeCommunicator.toPrimary('command:end', { id, name })
+    let serializedSubject = serialize(cy.state('subject'))
```

Comment:
It might be possible to avoid doing this if we aren't the last command in the queue. I'll investigate that.

Reply:
I could do this, is this crazy? We're getting the last command in the queue and comparing ids 
```suggestion
    let serializedSubject: string | undefined = undefined

    if (cy.queue.at(cy.queue.length - 1).get('id') === id) {
      serializedSubject = serialize(cy.state('subject'))
    }
```

Reply:
Do we ever need to yield something back that isn't the last command? I don't think it's crazy and it could save on serialization costs if other commands are yielding subjects.